script/install-nginx.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2018-2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Instaling Nginx"
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install -n nginx-ingress --create-namespace nginx-ingress ingress-nginx/ingress-nginx
+
+# Wait for load balancer to get the IP
+LB_IP=""
+while [ -z $LB_IP ]; do
+  echo "Waiting for external IP"
+  LB_IP=$(kubectl -n nginx-ingress get service nginx-ingress-ingress-nginx-controller --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}")
+  [ -z "$LB_IP" ] && sleep 10
+done
```

Comment:
Ideally we would like this to have a max retries limitation.

Reply:
yeah, I think we should add it to avoid potential headaches 😄

Reply:
There is the timeout of the pipeline run itself that will kick in for now. Will try to tackle it in a further PR.